EmerisHQ/demeris
# fix: fix the staking balance, refactored the getDisplayAmount into a pure function to utils/basic.ts

## Description
Fixes: #1328 

## Feature flags
?VUE_APP_FEATURE_STAKING=1

## Testing
1. Go to https://deploy-preview-1332--emeris-app.netlify.app/asset/uatom?VUE_APP_FEATURE_STAKING=1
2. View the staking component's binance validator and check whether it has an absurd amount of decimal places

---
